app/commands/virole.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * Execute on virole command
+ * @param {Discord.Message} msg - discord message
+ * @param {Array} args - passed arguments
+ */
+const virole = async (msg, args) => {
+    // get parameters
+    const givenRole = args[0];
+    const option = args[1];
+    const update = args[2];
+
+    // get role by name
+    let findRole = msg.guild.roles.cache.find(
+        (role) => role.name === givenRole
+    );
+
+    // store the permissions then use this to set permissions in the function.
+    let options;
+
+    // check if givenRole exists?
+    if (findRole) {
+        switch (option) {
+            // if user want to update the color
+            case "color":
+                if (update) {
+                    // This expression to verify the user only set a valid Hex number like #85514d
+                    if (new RegExp("#[0-9a-f]{6}").test(update)) {
```

Comment:
Can't change the color value when testing the following:
```bash
!sudo virole [role-name] color #FF0000
```
However, it works when testing the following:
```bash
!sudo virole [role-name] color #ff0000
```
I think updating the regular expression to consider capital hex values.

Reply:
Hello  
Thank you so much. 
I've fixed this bug in the below commit